__tests__/app.test.js

```diff
@@ -113,3 +113,71 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("should add a comment for an article", () => {
+    const newComment = {
+      username: "rogersop",
+      body: "example of comprehensive and non biased comment",
+    };
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send(newComment)
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment.body).toBe(newComment.body);
+      });
+  });
+  describe("shoud handle errors", () => {
+    test("should respond with 400 bad request when username is not provided", () => {
```

Comment:
You could make this more general and describe that this is any kind of malformed input body